packages/app/src/app/pages/Dashboard/Components/NewTeamModal/TeamCreate.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { useActions, useAppState } from 'app/overmind';
+import { Stack, Text, Link, Element, Icon } from '@codesandbox/components';
+import { InputText } from 'app/components/dashboard/InputText';
+import { StyledButton } from 'app/components/dashboard/Button';
+import { docsUrl } from '@codesandbox/common/lib/utils/url-generator';
+
+export const TeamCreate: React.FC<{ onComplete: () => void }> = ({
+  onComplete,
+}) => {
+  const { dashboard } = useAppState();
+  const actions = useActions();
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [existingTeamError, setExistingTeamError] = useState(false);
+
+  const onSubmit = async event => {
```

Comment:
this is just a clone of the TeamName component, but the submit form will create the team, while the other one used for the regular deploys will just rename the workspace after you go through stripe successfully